### ISSUE TITLE
feature request - option to change the menu button from "game exit" to interactive "quick menu"

### ISSUE THREAD

**Issue by ilmenit** [OnionUI/Onion#161]

**Comment by Olywa** [comment]
@Aemiii91 Request for Quick Menu overlay - I know Toto explored this for a while (in the village thread I think) but was unable to achieve satisfactory performance.

**Comment by pashque** [comment]
Yes, I totally agree ! It will be great to have a possibili to chose from a game switcher and more simple in game menu like in stock os. I like onion much for a many good features, but I honestly forced to use a stock one because of its simplicity and because for me a game switcher is over featured and I can not disable it. Can we get a simple stock-like menu for a Christmas’s, please ?

**Comment by r57zone** [comment]
I agree, this is a great solution.
